src/pt/prismascans/src/eu/kanade/tachiyomi/extension/pt/prismascans/DemonSect.kt

```diff
@@ -21,6 +22,14 @@ class DemonSect : Madara(
     override val id: Long = 8168108118738519332
 
     override val client: OkHttpClient = super.client.newBuilder()
+        .addInterceptor { chain ->
+            val response = chain.proceed(chain.request())
+            val request = response.request
+            if (request.url.toString().contains("login", ignoreCase = true)) {
```

Comment:
I think it's better to check the path segment instead of the string in the full url